packages/components/src/_components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,218 @@
+import { cloneElement, forwardRef, useId } from 'react'
+
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import {
+  ArrowRightIcon,
+  CheckIcon,
+  ExternalIcon,
+  SearchIcon,
+} from '@status-im/icons/20'
+import { cva, cx } from 'cva'
+
+import { Checkbox } from '../checkbox'
+import { Input } from '../input'
+
+import type { IconElement } from '../types'
+
+type Props = DropdownMenu.DropdownMenuProps & {
+  children: [React.ReactElement, React.ReactElement]
+}
+
+export const Root = (props: Props) => {
+  const { children, ...rootProps } = props
+
+  const [trigger, content] = children
+
+  return (
+    <DropdownMenu.Root {...rootProps}>
+      <DropdownMenu.Trigger asChild>{trigger}</DropdownMenu.Trigger>
+      {content}
+    </DropdownMenu.Root>
+  )
+}
+
+export const Content = forwardRef<
+  React.ElementRef<typeof DropdownMenu.Content>,
+  DropdownMenu.DropdownMenuContentProps
+>((props, ref) => {
+  const {
+    align = 'start',
+    side = 'bottom',
+    sideOffset = 4,
+    ...contentProps
+  } = props
+  return (
+    <DropdownMenu.Portal>
+      <DropdownMenu.Content
+        ref={ref}
+        align={align}
+        side={side}
+        sideOffset={sideOffset}
+        {...contentProps}
+        className={cx(
+          'w-64 rounded-12 border border-neutral-10 bg-white-100 p-1 shadow-3',
+          'dark:border-neutral-90 dark:bg-neutral-95',
+          props.className,
+        )}
+      />
+    </DropdownMenu.Portal>
+  )
+})
+
+Content.displayName = DropdownMenu.Content.displayName
+
+export const Search = (props: React.ComponentPropsWithoutRef<typeof Input>) => {
+  return (
+    <div className="mb-1 p-1">
+      <Input {...props} size="32" icon={<SearchIcon />} aria-label="Search" />
+    </div>
+  )
+}
+
+const itemStyles = cva({
+  base: [
+    'flex h-8 cursor-pointer select-none items-center gap-2 rounded-8 px-2 text-15 transition-colors active:bg-neutral-10',
+    'outline-none data-[highlighted]:bg-neutral-5',
+    'dark:active:bg-customisation-50/10 dark:hover:bg-customisation-50/5',
```

Comment:
Is there difference for `data-[highlighted]` and `hover`?

Reply:
yes, keyboard usage